wgengine/wgengine.go

```diff
@@ -59,7 +60,7 @@ type Engine interface {
 	// sends an updated network map.
 	//
 	// The returned error is ErrNoChanges if no changes were made.
-	Reconfig(cfg *wgcfg.Config, dnsDomains []string, localSubnets []wgcfg.CIDR, noSNAT bool) error
+	Reconfig(cfg *wgcfg.Config, routerCfg router.Settings) error
```

Comment:
little weird to see `Reconfig` five times on this line (`Reconfig`, `cfg`, `wgcfg`, `Config`, `routerCfg`), and then--surprise!- `Settings`. Especially since the variable of type `Settings` is called `routerCfg`.

Maybe it wants to be `router.Config` to match `wgcfg.Config`?

Reply:
Sure, seems like a harmless rename. Added a followup commit that does this.